tests/pkcli/ci1_test.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""test ci
+
+:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pkg_resources
+from pykern.pkdebug import pkdp
+import re
+import pytest
+
+
+def test_check_prints_dot_prefix():
+    from pykern.pkcli import ci, test
+    from pykern import pkunit
+    from pykern import pksetup
+
+    ci._EXCLUDE_FILES = re.compile(
+        f"/{test.SUITE_D}/.*{pkunit.DATA_DIR_SUFFIX}/"
+        + f"|/{pksetup.PACKAGE_DATA}/"
+        + r"|pkdebug[^/]*\.py$"
+        + r"|\.*/"
+    )
+
+    for d in pkunit.case_dirs("ignore_dot_prefix"):
```

Comment:
I think having separate case_dirs causes confusion. Just one directory with all the different cases with mistakes in them that all should be ignored.

Reply:
Wont having a separate test.py necessitate a separate _data dir?

Reply:
Have one test (ci1_test) with one case dir (ci1_data/1.in).